mmv1/products/compute/BackendService.yaml

```diff
@@ -261,6 +270,39 @@ properties:
             Used when balancingMode is UTILIZATION. This ratio defines the
             CPU utilization target for the group. Valid range is [0.0, 1.0].
           default_from_api: true
+        - name: 'customMetrics'
+          type: Array
+          description: |
+            The set of custom metrics that are used for <code>CUSTOM_METRICS</code> BalancingMode.
+          is_set: false
```

Comment:
This should be the default, so it's not needed

Reply:
Done.